src/routers/authentication.ts

```diff
@@ -28,7 +28,7 @@ const router = createRouter();
        token : eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJmM2YxYTNiZjk3OGU0MNDVkY2Y2YSJ9.TLkEYpBlKrDxNO73edz-bXJ6XH1rqx8LimtkY
```

Comment:
does this file need to be renamed to authenticate? and don't you need to change the route name in routers/index.ts?